sway/desktop/output.c

```diff
@@ -210,22 +239,32 @@ static void output_frame_notify(struct wl_listener *listener, void *data) {
 		}
 	}
 
+	// TODO: Consider revising this when fullscreen windows are supported
```

Comment:
`render_surface` should also be called with `&now` instead of `&soutput->last_frame`. Same for `wlr_output_swap_buffers`.